roles/contiv_network/tasks/main.yml

```diff
@@ -1,10 +1,25 @@
 ---
 # This role contains tasks for configuring and starting netmaster and netplugin service
 
+- name: remove older netmaster
```

Comment:
how about a m5sum? But I agree this would easier if our binaries support a version flag.

I would still avoid these remove task as a workaround for some missing function,as I fear we will start depending on it soon and it will be hard to remove later.

What is the usecase you are trying to solve by removing the binaries? Can we use the fact that a new binary was indeed downloaded by looking at the result of the `get_url` result?